lib/apiservers/portlayer/swagger.json

```diff
@@ -2095,6 +2204,17 @@
 				}
 			}
 		},
+		"KeyValue": {
+			"type": "object",
+			"properties": {
+				"Key": {
+					"type": "string"
+				},
+				"Value":{
+					"type": "string"
```

Comment:
Is this value encoded in some fashion, e.g. base64? Was expecting something like:
```
"name": "kvValue",
"in": "body",
"schema": {
	"type": "string",
	"format": "binary"
}
```